sound/soc/sof/Kconfig

```diff
@@ -52,6 +52,9 @@ config SND_SOC_SOF_DEVELOPER_SUPPORT
 	  Say Y if you are involved in SOF development and need this option
```

Comment:
I don't know how you format your commit messages but you can use more words per line...